examples/push-file

```diff
@@ -15,76 +16,65 @@ def main():
 
     parser = argparse.ArgumentParser(
         description="Upload a file to an Amazon S3 bucket and publish data"
-                    " about it in a DynamoDB table."
+        " about it in a DynamoDB table."
     )
     parser.add_argument(
-        "--file-path",
-        required=True,
-        help="Local filesystem path to the file."
+        "--file-path", required=True, help="Local filesystem path to the file."
     )
     parser.add_argument(
         "--file-metadata",
         default={},
         type=json.loads,
         help="Additional data sent with the file in JSON string format,"
-             " e.g., '{\"key\": \"value\"}'"
+        ' e.g., \'{"key": "value"}\'',
     )
```

Comment:
For example, --file path was collapsed to a single line and the quotation usage in --file-metadata's help message continuation is now inconsistent. These changes aren't particularly valuable so I think it's best to leave things how they are if Black will abide. Things like this, while minor, can distract reviewers from more important changes and make the review process more difficult.

Reply:
> Did Black do a lot of formatting? 

Yes. Black is responsible for all of the formatting changes.